modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -98,14 +114,18 @@ centos8-uyuni-client centos8-appstream
 ====
 The client tools channel provided by [command]``spacewalk-common-channels`` is sourced from {uyuni} and not from {suse}.
 ====
+endif::[]
 
 
+There are two ways to check if a channel has finished synchronizing:
 
-.Procedure: Synchronizing {centos} repositories
-
-. In the {productname} {webui}, navigate to menu:Software[Manage], and check every {centos} channel.
-. In the [guimenu]``Repositories`` tab, navigate to the [guimenu]``Sync`` subtab, and click btn:[Sync Now].
-You can also create a regular synchronization schedule on this page.
+* In the {productname} {webui}, navigate to menu:Admin[Setup Wizard] and select the [guimenu]``SUSE Products`` tab.
++
+This dialog displays a completion bar for each product when they are being synchronized.
+* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log`` file.
```

Comment:
```suggestion
* Check the synchronization log file at the command prompt:
+
----
tail -f /var/log/rhn/reposync/channel-label.log
----
```
And do not forget to explain that "channel-label" is a replaceable.

Reply:
updated in all of them.